litmus-portal/graphql-server/pkg/chaos-workflow/handler/handler.go

```diff
@@ -168,6 +168,22 @@ func QueryWorkflowRuns(input model.GetWorkflowRunsInput) (*model.GetWorkflowsOut
 		{"isRemoved", 1},
 	}
 
+	// Filter the available workflows where isRemoved is false
+	matchWfRemovedStage := bson.D{
```

Comment:
Why are we moving this out from the if condition?

Reply:
This is a mandatory stage, we will filter out the workflow runs that are removed, or in other way, we'll only include workflow runs whose isRemoved state is false